src/main/java/com/enterprise/eventmanagementsystem/dao/EventDAO.java

```diff
@@ -6,6 +6,11 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * This dao allows access and maps many of the various functions such as specifically
+ * fetching events by their id's.
+ * @author Ali Ibrahim, Benjamin Gomori, Kevin Zhu, and Son Bo
```

Comment:
@author tag is deprecated, now that we can see commit history and author in GitHub.
Believe it or not, that used to not be possible.